<ui version="4.0">
 <class>EmailSenderWindow</class>
 <widget class="QWidget" name="EmailSenderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>대량 메일 발송기</string>
  </property>

  <!-- 발송자 이메일 주소 입력 -->
  <widget class="QLabel" name="labelSenderEmail">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>발송자 이메일</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSenderEmail">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>

  <!-- 발송자 비밀번호 입력 -->
  <widget class="QLabel" name="labelSenderPassword">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>발송자 비밀번호</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSenderPassword">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>

  <!-- 엑셀 파일 업로드 버튼 -->
  <widget class="QPushButton" name="btnUploadExcel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>엑셀 파일 업로드</string>
   </property>
  </widget>

  <!-- 엑셀 데이터 테이블 -->
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>560</width>
     <height>200</height>
    </rect>
   </property>
  </widget>

  <!-- 메일 발송 버튼 -->
  <widget class="QPushButton" name="btnSendEmails">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>메일 발송 시작</string>
   </property>
  </widget>

  <!-- 로그 출력창 -->
  <widget class="QTextEdit" name="textEditLog">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>560</width>
     <height>100</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>로그 출력창</string>
   </property>
  </widget>

  <!-- 로그 엑셀 저장 -->
  <widget class="QPushButton" name="btnSaveLogToExcel">
 <property name="geometry">
  <rect>
   <x>200</x>
   <y>370</y>
   <width>150</width>
   <height>30</height>
  </rect>
 </property>
 <property name="text">
  <string>로그 엑셀로 저장</string>
 </property>
</widget>


  <!-- 로그 초기화 버튼 -->
  <widget class="QPushButton" name="btnClearLog">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>370</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>로그 초기화</string>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>
